transposon/mapIntegrations.sh

```diff
@@ -106,16 +101,30 @@ if [ "$(zcat -f $f1 | head -n 4000 | awk 'NR % 4 == 2 { sum += length($0) }; END
     bwaAlnExtractOpts="-n 3 -r @RG\\tID:${sample}\\tLB:$DS\\tSM:${DS_nosuffix}"
     extractcmd="samse ${bwaAlnExtractOpts} ${bwaIndex} $TMPDIR/${sample}.genome.sai $TMPDIR/${sample}.genome.fastq.gz"
 else
-    # Paired-end mapping using bwa mem
+    echo "Paired-end mapping using bwa mem"
     echo "Adaptive trimming of R1"
+    
+    # R1 read adapters
+    #BUGBUG move this so it isn't hard coded
+    altDpnSeq="GATCTTTGTCCAAACTCATCGAGCTCGG"
+    R2PrimerSeq=$(echo $plasmidSeq | tr "[ATGC]" "[TACG]" | rev)
+    # R2 read adapters
+    firstDpnRevSeq=$(echo ${BCreadSeq} | awk '{ print substr($0, 0, length($0)-4) }' | tr "[ATGCB]" "[TACGN]" | rev)
+    altDpnRevSeq=$(echo ${altDpnSeq} | tr "[ATGC]" "[TACG]" | rev)
+    
     zcat -f $f1 |
     awk -v firstline=$firstline -v lastline=$lastline 'NR>=firstline && NR<=lastline' |
     cutadapt -Z -j $NSLOTS -o $TMPDIR/${sample}.R1.fastq.gz -u ${R1primerlen} -a ${R2PrimerSeq} -g X${altDpnSeq} -
     
     echo "Trimming $R2primerlen bp primer from R2"
     zcat -f $f2 | 
     awk -v firstline=$firstline -v lastline=$lastline 'NR>=firstline && NR<=lastline' |
-    cutadapt -Z -j $NSLOTS -u 18 -a ${firstDpnRevSeq} -a ${altDpnRevSeq} -o $TMPDIR/${sample}.genome.fastq.gz -
+    cutadapt -j $NSLOTS -u 18  -o - - |
```

Comment:
Andre, why is this hardcoded as 18?

Reply:
It might be something I missed, it should be `${R1primerlen}` instead.

Reply:
You mean ${R2primerlen}?

Reply:
Yes, it should be `${R2primerlen}`